generator/src/test/java/com/scottlogic/deg/generator/cucumber/Before.feature

```diff
@@ -66,8 +66,7 @@ Scenario: Running a 'before' request that specifies an invalid leap year should
        Then the profile is invalid
      And no data is created
 
-#before
-@ignore #667 & 594, granularity should be millisecond by default and generation should be descending when upper-bound only is provided
+@ignore #594 generation should be descending when upper-bound only is provided
```

Comment:
Would be best to keep the comment `#before` as this delineates sections of scenarios